packages/ix-player/src/index.ts

```diff
@@ -355,6 +360,14 @@ class IxPlayerElement extends VideoApiElement {
 
   #monitorLiveWindow() {
     this.mediaController?.addEventListener('mediaplayrequest', (event) => {
+      if ((event.target as Element)?.id === 'retry-btn' && this.retries) {
+        this.retries = 0;
+        const { src } = this;
+        if (src !== undefined) {
+          this.src = '';
+          this.src = src;
+        }
+      }
```

Comment:
im not sure if this is the correct method to add or if we should create another method, but we need an event listener for clearing errors when metadata is loaded, example I have an error message when my video has loaded the retry screen can be removed if we added a eventlistener that removes error messaging. 

Sudo logic:
Run retries
If no more retries and last error was 423
Set eventlistener to remove error when loadeddata happens

<img width="569" alt="Screen Shot 2023-10-19 at 5 14 58 PM" src="https://github.com/imgix/ix-elements/assets/13306885/86a53c17-5cde-4191-930b-7a10a2eaec65">

Reply:
comment: this is a good call; definitely best to clear the error on a successful metadata load

discussion: maybe the best place to listen for this is in a new private method `#monitorMetadataLoad` than can add and remove the event listener, and define the handler. Make me think we should also have a method for listening for the retry click, since it's kinda shoehorned into `#monitorHover` at the moment.